### ISSUE TITLE
how to get content of one single file from directory with type file not raw

### ISSUE THREAD

**Issue by keinagae** [web3-storage/ipfs-car#136]

**Comment by olizilla** [comment]
@keinagae are you seeing an error fetching something from w3s.link or another IPFS gateway? Can you share more information about what you've tried and what didn't work?

In general you should be able to add the path on to the root CID for any files / directories in your DAG like `https://<root CID>.ipfs.w3s.link/path/to/thing`

**Comment by olizilla** [comment]
Ah, missed that this is on the `ipfs-car` repo... ok. let me try again.

`ipfs-car` doens't currently support extracting a single file from a directory, but that is a good feature request!
